approval_polls/views.py

```diff
@@ -79,7 +79,7 @@ def add(request):
     if not len(choices):
       return render(request, 'approval_polls/error.html', {'error_message': 'At least one choice is required'})
 
-    p = Poll(question=question, pub_date=timezone.now())
+    p = Poll(question=question, pub_date=timezone.now(), username=request.POST['username'].strip())
```

Comment:
The username should be found by using the `request.session` object. For a detailed explanations on sessions see [the documentation](https://docs.djangoproject.com/en/1.8/topics/http/sessions/)